### DISCUSSION TITLE
I can't build this in VS2022

### DISCUSSION THREAD

**Discussion by mburns08109** [rubberduck-vba/Rubberduck#5968] [Q&A]
When I attempt to build this in VS2022 as-is, I get the following failure result:
C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(2301,5): warning MSB3270: There was a mismatch between the processor architecture of the project being built "MSIL" and the processor architecture of the reference "olewoo", "x86". This mismatch may cause runtime failures. Please consider changing the targeted processor architecture of your project through the Configuration Manager so as to align the processor architectures between your project and references, or take a dependency on references with a processor architecture that matches the targeted processor architecture of your project.

Since this is a result of trying to compile a nuget component (or what used to be a nuget component), I have no idea how to address this.

=== Build Results output ===
Build started...
Restored C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.InternalApi\Rubberduck.InternalApi.csproj (in 66 ms).
Restored C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.VBEEditor\Rubberduck.VBEditor.csproj (in 13 ms).
Restored C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.VBEditor.VBA\Rubberduck.VBEditor.VBA.csproj (in 11 ms).
Restored C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.VBEditor.VB6\Rubberduck.VBEditor.VB6.csproj (in 15 ms).
Restored C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.Deployment\Rubberduck.Deployment.csproj (in 74 ms).
Restored C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.Parsing\Rubberduck.Parsing.csproj (in 68 ms).
Restored C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.Main\Rubberduck.Main.csproj (in 74 ms).
Restored C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.CodeAnalysis\Rubberduck.CodeAnalysis.csproj (in 13 ms).
Restored C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.Core\Rubberduck.Core.csproj (in 15 ms).
Restored C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.Refactorings\Rubberduck.Refactorings.csproj (in 11 ms).
Restored C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.Interaction\Rubberduck.Interaction.csproj (in 8 ms).
Restored C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.UnitTesting\Rubberduck.UnitTesting.csproj (in 17 ms).
Restored C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.SettingsProvider\Rubberduck.SettingsProvider.csproj (in 15 ms).
Restored C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.SmartIndenter\Rubberduck.SmartIndenter.csproj (in 13 ms).
Restored C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\RubberduckTests\RubberduckTests.csproj (in 17 ms).
Restored C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.Resources\Rubberduck.Resources.csproj (in 10 ms).
Restored C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.RegexAssistant\Rubberduck.RegexAssistant.csproj (in 8 ms).
1>------ Build started: Project: Rubberduck.Main, Configuration: Debug Any CPU ------
1>Missing the required meta projects. Building the meta projects...
1>Determining projects to restore...
1>Restored C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\RubberduckCodeAnalysis\RubberduckCodeAnalysis.csproj (in 15.28 sec).
1>Determining projects to restore...
1>Restored C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.Deployment.Build\Rubberduck.Deployment.Build.csproj (in 1 ms).
1>Determining projects to restore...
1>Restored C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\RubberduckTestsCodeAnalysis\RubberduckTestsCodeAnalysis.csproj (in 3.34 sec).
1>1 of 2 projects are up-to-date for restore.
1>RubberduckCodeAnalysis -> C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\RubberduckCodeAnalysis\bin\RubberduckCodeAnalysis.dll
1>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(2301,5): warning MSB3270: There was a mismatch between the processor architecture of the project being built "MSIL" and the processor architecture of the reference "olewoo", "x86". This mismatch may cause runtime failures. Please consider changing the targeted processor architecture of your project through the Configuration Manager so as to align the processor architectures between your project and references, or take a dependency on references with a processor architecture that matches the targeted processor architecture of your project.
1>C:\Program Files\Microsoft Visual Studio\2022\Professional\MSBuild\Current\Bin\amd64\Microsoft.Common.CurrentVersion.targets(2301,5): warning MSB3270: There was a mismatch between the processor architecture of the project being built "MSIL" and the processor architecture of the reference "olewoo_interop", "x86". This mismatch may cause runtime failures. Please consider changing the targeted processor architecture of your project through the Configuration Manager so as to align the processor architectures between your project and references, or take a dependency on references with a processor architecture that matches the targeted processor architecture of your project.
1>Rubberduck.Deployment.Build -> C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.Deployment.Build\bin\Rubberduck.Deployment.Build.dll
1>Done building project "Rubberduck.Deployment.Build.csproj".
1>RubberduckTestsCodeAnalysis -> C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\RubberduckTestsCodeAnalysis\bin\RubberduckTestsCodeAnalysis.dll
1>Built the meta projects
1>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.CodeAnalysis\bin\Debug\net461\Rubberduck.CodeAnalysis.dll' could not be found
1>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.Core\bin\Debug\net461\Rubberduck.Core.dll' could not be found
1>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.Interaction\bin\Debug\net461\Rubberduck.Interaction.dll' could not be found
1>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.InternalApi\bin\Debug\net461\Rubberduck.InternalApi.dll' could not be found
1>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.Parsing\bin\Rubberduck.Parsing.dll' could not be found
1>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.Refactorings\bin\Debug\net461\Rubberduck.Refactorings.dll' could not be found
1>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.RegexAssistant\bin\Debug\net461\Rubberduck.RegexAssistant.dll' could not be found
1>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.Resources\bin\Debug\net461\Rubberduck.Resources.dll' could not be found
1>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.SettingsProvider\bin\Debug\net461\Rubberduck.SettingsProvider.dll' could not be found
1>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.SmartIndenter\bin\Debug\net461\Rubberduck.SmartIndenter.dll' could not be found
1>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.UnitTesting\bin\Debug\net461\Rubberduck.UnitTesting.dll' could not be found
1>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.VBEEditor\bin\Debug\net461\Rubberduck.VBEditor.dll' could not be found
1>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.VBEditor.VB6\bin\Debug\net461\Rubberduck.VBEditor.VB6.dll' could not be found
1>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.VBEditor.VBA\bin\Debug\net461\Rubberduck.VBEditor.VBA.dll' could not be found
1>Done building project "Rubberduck.Main.csproj" -- FAILED.
2>------ Build started: Project: RubberduckTests, Configuration: Debug Any CPU ------
2>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.CodeAnalysis\bin\Debug\net461\Rubberduck.CodeAnalysis.dll' could not be found
2>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.Core\bin\Debug\net461\Rubberduck.Core.dll' could not be found
2>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.Main\bin\Debug\net461\Rubberduck.dll' could not be found
2>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.Interaction\bin\Debug\net461\Rubberduck.Interaction.dll' could not be found
2>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.InternalApi\bin\Debug\net461\Rubberduck.InternalApi.dll' could not be found
2>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.Parsing\bin\Rubberduck.Parsing.dll' could not be found
2>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.Refactorings\bin\Debug\net461\Rubberduck.Refactorings.dll' could not be found
2>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.RegexAssistant\bin\Debug\net461\Rubberduck.RegexAssistant.dll' could not be found
2>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.Resources\bin\Debug\net461\Rubberduck.Resources.dll' could not be found
2>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.SettingsProvider\bin\Debug\net461\Rubberduck.SettingsProvider.dll' could not be found
2>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.SmartIndenter\bin\Debug\net461\Rubberduck.SmartIndenter.dll' could not be found
2>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.UnitTesting\bin\Debug\net461\Rubberduck.UnitTesting.dll' could not be found
2>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.VBEEditor\bin\Debug\net461\Rubberduck.VBEditor.dll' could not be found
2>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.VBEditor.VB6\bin\Debug\net461\Rubberduck.VBEditor.VB6.dll' could not be found
2>CSC : error CS0006: Metadata file 'C:\Users\MXB0310\Source\Repos\rubberduck-vba\rubberduck\Rubberduck.VBEditor.VBA\bin\Debug\net461\Rubberduck.VBEditor.VBA.dll' could not be found
2>Done building project "RubberduckTests.csproj" -- FAILED.
========== Build: 0 succeeded, 2 failed, 15 up-to-date, 0 skipped ==========

**Comment by Vogel612** [comment]
moved to issues, since our chatbot does not watch discussions.
